fuzzypp/src/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Build static library for use by CLI & tests.
+add_library(${FUZZYPP_LIB_NAME} STATIC
+            cliopts.cpp
+            preprocessor.cpp
+            "${CMAKE_SOURCE_DIR}/external/simplecpp/simplecpp.cpp")
+
+target_include_directories(${FUZZYPP_LIB_NAME} PUBLIC
+                           "${CMAKE_SOURCE_DIR}/external/cxxopts/include"
+                           "${CMAKE_SOURCE_DIR}/external/simplecpp")
```

Comment:
Just a comment, no action needed.

There is an option (`SYSTEM`) to mark header search paths as system headers, this way we won't get any warnings that are coming from third-party libraries. Whether it is desired or not is a separate question though 😄